cdc-accelerator/feature/partner/__tests__/permissionGroups.test.js

```diff
@@ -43,6 +44,24 @@ describe('Permission Groups test suite', () => {
                 JSON.stringify(expectedPermissionGroupsResponseAfterRemovingBuiltInGroups, null, 4),
             )
         })
+        test('PermissionGroups should not write empty groups', async () => {
+            axios.mockResolvedValue({ data: expectedPermissionGroupsResponseWithEmptyGroup })
+            const getSiteInfo = {
+                partnerId: 123123,
+                dataCenter: 'eu1',
+            }
+
+            fs.existsSync.mockReturnValue(false)
+            fs.mkdirSync.mockReturnValue(undefined)
+            fs.writeFileSync.mockReturnValue(undefined)
+            await permissionGroups.init(partnerBaseDirectory, getSiteInfo)
+            const srcDirectory = path.join(partnerBaseDirectory, permissionGroups.getName())
+            expect(fs.existsSync).toHaveBeenCalledWith(srcDirectory)
```

Comment:
It is expected fs.writeFile to be called?!
please also expect acl.init has not been called also